### PR TITLE
Move accountdb schema migration to trackerdb.go

### PR DESCRIPTION
**Pull request by algorandskiy**
## Summary

* Moved code from **acctupdates.go** to **trackerdb.go**.
* Extracted  db creation and init from`accountsInitialize` to `trackerDBInitialize`
* `reloadLedger` now calls `trackerDBInitialize` right after block db init
* `reloadLedger` also has post-creation actions like vacuuming

Limitations:
1. Tables creation still calls `accountsInit` for initial tables creation and inserting genesis accounts.
2. Migration code still have some business logic. 

## Test Plan

Use existing test suite

### CODE REVIEW

**Comment by algorandskiy** [comment]
Not sure why it started the Branch build

File: ledger/acctupdates.go [hunk 1]
```diff
@@ -1164,13 +1137,10 @@ func (au *accountUpdates) initializeFromDisk(l ledgerForTracker) (lastBalancesRo
 		return
 	}
 
-	// the VacuumDatabase would be a no-op if au.vacuumOnStartup is cleared.
-	au.vacuumDatabase(context.Background())
+	au.accountsq, err = accountsDbInit(au.dbs.Rdb.Handle, au.dbs.Wdb.Handle)
```
Comment (tsachiherman) [inline ledger/acctupdates.go:1140]: we really could use a better name for this method. It doesn't initializing anything with the db rather then preparing precompiled sql statements.

File: ledger/trackerdb.go [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+// The accountbase would get initialized with the au.initAccounts
+// The accounttotals would get initialized to align with the initialization account added to accountbase
+// The acctrounds would get updated to indicate that the balance matches round 0
+//
+func (tu *trackerDbSchemaInitializer) upgradeDatabaseSchema0(ctx context.Context, tx *sql.Tx) (err error) {
+	tu.log.Infof("accountsInitialize initializing schema")
+	tu.newDatabase, err = accountsInit(tx, tu.initAccounts, tu.initProto)
+	if err != nil {
+		return fmt.Errorf("accountsInitialize unable to initialize schema : %v", err)
+	}
```
Comment (tsachiherman) [inline ledger/trackerdb.go:209]: change error/info messages accountsInitialize  -> upgradeDatabaseSchema0

**Review by tsachiherman — CHANGES_REQUESTED** [review]
please review error/info messages and update the method names; looks good otherwise.

File: ledger/acctupdates.go (hunk 1 above)
Comment (algorandskiy) [reply ledger/acctupdates.go]: done

File: ledger/trackerdb.go (hunk 2 above)
Comment (algorandskiy) [reply ledger/trackerdb.go]: done

**Pull request merged**